core/src/main/java/nl/nn/adapterframework/http/HttpSenderBase.java

```diff
@@ -471,9 +461,13 @@ public CloseableHttpClient getHttpClient() {
 		return httpClient;
 	}
 
-	public void close() {
-		connectionManager.shutdown();
-		connectionManager = null;
+	public void close() throws SenderException {
+		try {
+			//Close the HttpClient and ConnectionManager to release resources and potential open connections
+			getHttpClient().close();
```

Comment:
Is het niet meer nodig om de ConnectionManager zelf te closen?

Reply:
Nee, zie: https://hc.apache.org/httpcomponents-client-4.5.x/tutorial/html/connmgmt.html